en/config_vtol/README.md

```diff
@@ -1,3 +1,11 @@
 # VTOL Configuration
 
-This section contains topics related to VTOL configuration and tuning.
\ No newline at end of file
+First perform the [Standard Configuration](../config/README.md).
+As part of this you should calibrate the [Airspeed sensor](../config/airspeed.md) (optional, but highly recommended) and [assign a VTOL transition switch](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set) to your RC controller.
+
+Then perform VTOL-specific configuration and tuning:

```

Comment:
@sfuhrer There are 4 config docs below. The first one is clearly for Quadplane. Can you confirm whether/not the other three apply to quadplane, tiltrotor, tailsitter?